File: atcoder.jp/contests/ddcc2020-qual/tasks/ddcc2020_qual_a/main.cpp
#include <bits/stdc++.h>
using namespace std;

int additional_prize(int x, int y) {
  if (x == 1 && y == 1) return 400000;
  return 0;
}

int main() {
  int X, Y;
  cin >> X >> Y;

  map<int, int> prize = {{3, 100000}, {2, 200000}, {1, 300000}};
  int ans = prize[X] + prize[Y] + additional_prize(X, Y);

  cout << ans << endl;
}

File: atcoder.jp/contests/ddcc2020-qual/tasks/ddcc2020_qual_a/main.hs
prize :: Int -> Int
prize n = case n of
  3 -> 100000
  2 -> 200000
  1 -> 300000
  _ -> 0

additionalPrize :: Int -> Int -> Int
additionalPrize x y
  | x == 1 && y == 1 = 400000
  | otherwise = 0

main :: IO ()
main = do
  [x, y] <- map read . words <$> getLine

  let ans = prize x + prize y + additionalPrize x y
  print ans
